Gruntfile.js

```diff
@@ -1663,6 +1663,7 @@ module.exports = function(grunt) {
 	grunt.registerTask( 'verify:source-maps', function() {
 		const ignoredFiles = [
 			'build/wp-includes/js/dist/components.js',
+			'build/wp-includes/js/dist/data.js',
```

Comment:
I think this should be okay; the `sourceMappingURL` in this package build doesn't cause 404 errors.

The recent update of the Redux package might have introduced the `sourceMappingURL` change. See: https://github.com/WordPress/gutenberg/pull/66966.